### PR TITLE
upgrade to play 2.7.3

### PR DESCRIPTION
**Pull request by schmitch**
also needs to be backported.

### CODE REVIEW

**Comment by mkurz** [comment]
Nice catch.

**Comment by schmitch** [comment]
Looks like we need to update to sbt 1.2.8 first.. some nasty resolution errors

File: build.sbt [hunk 1]
```diff
@@ -23,15 +22,24 @@ lazy val commonSettings = mimaDefaultSettings ++ Seq(
...
 )
 
 // needs to be kept in sync with travis-ci
-val PlayVersion = playVersion(sys.env.getOrElse("PLAY_VERSION", "2.7.0"))
+val PlayVersion = playVersion(sys.env.getOrElse("PLAY_VERSION", "2.7.3"))
 
 // Version used to check binary compatibility
 val mimaPreviousArtifactsVersion = "7.0.0"
 
+def mimePreviousVersionExcludeScala213(modules: Set[ModuleID]): Set[ModuleID] = {
```
Comment (mkurz) [inline build.sbt]: @schmitch That should be an `a` and the end of `mime...` or not?

**Comment by schmitch** [comment]
I did it, if somebody wants to approve, we could merge it.

**Comment by marcospereira** [comment]
7.0.x backport: #167.

**Pull request merged**